vllm/lora/ops/bgmv_expand.py

```diff
@@ -167,3 +167,8 @@ def bgmv_expand(
         **config,
     )
     return
+
+
+bgmv_expand = torch.library.custom_op("lora::bgmv_expand",
```

Comment:
do you need to register the fake implementation?

Reply:
dynamo never complained about missing fake implementations.  although it seems like they should have them.